azure-storage-blob/azure/storage/blob/baseblobservice.py

```diff
@@ -1534,6 +1534,11 @@ def set_blob_service_properties(
         :type static_website:
             :class:`~azure.storage.common.models.StaticWebsite`
         '''
+        if all(parameter is None for parameter in [logging, hour_metrics, minute_metrics, cors, target_version, timeout,
```

Comment:
timeout shouldn't be part of the list

Reply:
since it just specifies the service timeout (not a property)

Reply:
This is addressed in the third commit..